test-utils/src/monerod/download.rs

```diff
@@ -24,23 +24,23 @@ static DOWNLOAD_MONEROD_MUTEX: Mutex<()> = Mutex::const_new(());
 /// Returns the file name to download and the expected extracted folder name.
 fn file_name(version: &str) -> (String, String) {
     let download_file = match (OS, ARCH) {
-        ("windows", "x64") | ("windows", "x86_64") => format!("monero-win-x64-{}.zip", version),
-        ("windows", "x86") => format!("monero-win-x86-{}.zip", version),
-        ("linux", "x64") | ("linux", "x86_64") => format!("monero-linux-x64-{}.tar.bz2", version),
-        ("linux", "x86") => format!("monero-linux-x86-{}.tar.bz2", version),
-        ("macos", "x64") | ("macos", "x86_64") => format!("monero-mac-x64-{}.tar.bz2", version),
+        ("windows", "x64" | "x86_64") => format!("monero-win-x64-{version}.zip"),
+        ("windows", "x86") => format!("monero-win-x86-{version}.zip"),
+        ("linux", "x64" | "x86_64") => format!("monero-linux-x64-{version}.tar.bz2"),
+        ("linux", "x86") => format!("monero-linux-x86-{version}.tar.bz2"),
+        ("macos", "x64" | "x86_64") => format!("monero-mac-x64-{version}.tar.bz2"),
```

Comment:
This is turning:
```
("windows", "x64") | ("windows", "x86_64")
```
into:
```
("windows", "x64" | "x86_64")
```
I didn't know I could like Rust's pattern matching even more :D

Reply:
Wow yeah TIL